### ISSUE TITLE
Doesn't seem to work any longer, as of the last few weeks.

### ISSUE THREAD

**Issue by prittjr** [mamuso/close-figma-tab#4]
I don't have much to add here, not sure what may have changed, but it isn't closing these tabs anymore.

**Comment by enricospada** [comment]
I'm having the same issue...the extension seems to require permission every time...
<img width="285" alt="image" src="https://github.com/mamuso/close-figma-tab/assets/12905003/93276b79-4ee0-4cb4-b5c6-a070c860c8ff">

**Comment by prittjr** [comment]
good catch, yes I'm seeing the same. I have no idea what's causing that; I've restarted chrome, removed and re-added the extension, etc and it doesn't help. Other extensions I use are correctly acting on their permission. I even downloaded these files from this repo and installed them, unaltered, as unpacked and it worked correctly then.
